### ISSUE TITLE
transparent background on charts

### ISSUE THREAD

**Issue by statgreen** [statisticssweden/PxWeb#183]
I have mailed pc-axis at scb on this issue before, and now raise it here, not to be forgotten.

For easy reuse of charts from saved queries in presentations I would very much like the possiblity to have a transparent background or some degree of background transparency.

Especially when a chart is placed upon a picture, this would make a BIG impact on the presentation - and a million times more presentable.

**Comment by statgreen** [comment]
yes - looking so much forward to try it out :-) When do you expect it to be on your beta-site?

**Comment by likp** [comment]
We currently do not have any beta site set up yet. We only have a demo site for the latest release of PxWeb. But I will try to arrange a beta site or a site that builds off of  the master branch.

**Comment by larpSTATGL** [comment]
I finally got a beta version running and have done the first couple of test. The transparent background will enable me to combine augmented reality Adobe aero movies/pics, just added a simple example. So thank you so much for this. But I will have an extra wish. Namely that the 2 ‘background color chart’ and ‘chart background alpha’ can be set in chart settings and kept in a saved query. The 2 settings could then be set individually. I only found a way in admin to change as general setting, which is too general.

![IMG_4022](https://user-images.githubusercontent.com/14994906/230755484-6abcaa17-1883-46db-9949-7a4275d68f4a.png)
